'unit_tests' can't pass building

Dear all,
I have downloaded the latest version of mfem from here and try to build the whole project by VS2017 in windows 10. I used the cmake to generate VS project (64bit) and didn't modify any default settings. When build the whole solution all projects passed except the one 'unit_tests'. The errors are like the following:
test_quadraturefunc.cpp
3>c:\mfem3.3\config/config.hpp(23): fatal error C1083: Cannot open include file: '_config.hpp': No such file or directory


In addition, I am trying to create a single 'tesla' solution, I added source files 'tesla.cpp' and 'tesla_solver.cpp' into the project, and set include directories where header files located, and included the 'mfem.lib' into linker input. When building this solution, the same errors displayed:
Cannot open include file: '_config.hpp': No such file or directory
tesla.cpp
1>c:\mfem3.3\config\config.hpp(23): fatal error C1083: Cannot open include file: '_config.hpp': No such file or directory

Could anyone help me to take a look at it?

Thanks,
Tang Laoya

--- a/tests/unit/CMakeLists.txt
+++ b/tests/unit/CMakeLists.txt
@@ -9,11 +9,11 @@
 # terms of the GNU Lesser General Public License (as published by the Free
 # Software Foundation) version 2.1 dated February 1999.
 
-# Include the build directory where mfem.hpp and mfem-performance.hpp are.
-include_directories(BEFORE ${PROJECT_BINARY_DIR})
 # Include the top mfem source directory - needed by some tests, e.g. to
 # #include "general/text.hpp".
 include_directories(BEFORE ${PROJECT_SOURCE_DIR})
+# Include the build directory where mfem.hpp and mfem-performance.hpp are.
+include_directories(BEFORE ${PROJECT_BINARY_DIR})
 # Include the source directory for the unit tests - catch.hpp is there.
 include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR})
 

